rbac/common/sawtooth/batcher.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2018 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+import logging
+from hashlib import sha512
+from uuid import uuid4
+from sawtooth_rest_api.protobuf import batch_pb2
+from sawtooth_rest_api.protobuf import client_batch_submit_pb2
+from sawtooth_rest_api.protobuf import transaction_pb2
+from rbac.addressing import addresser
+from rbac.transaction_creation.protobuf import rbac_payload_pb2
+from rbac.app.config import BATCHER_KEY_PAIR
+
+LOGGER = logging.getLogger(__name__)
+
```

Comment:
Add high level documentation (docstrings). This can wait until next PR in the queue